axum-core/src/response/into_response.rs

```diff
@@ -119,9 +119,7 @@ use std::{
 ///
 /// // `MyBody` can now be returned from handlers.
 /// let app = Router::new().route("/", get(|| async { MyBody }));
-/// # async {
-/// # hyper::Server::bind(&"".parse().unwrap()).serve(app.into_make_service()).await.unwrap();
-/// # };
+/// # let _: Router = app;
```

Comment:
I guess that answers my question 😄

Reply:
Hehe yes. I ran a quick search and replace to catch at least some of them.